p2panda-rs/src/operation/operation_value.rs

```diff
@@ -1,69 +1,44 @@
 // SPDX-License-Identifier: AGPL-3.0-or-later
 
-use serde::{Deserialize, Serialize};
-
 use crate::document::{DocumentId, DocumentViewId};
-use crate::hash::HashError;
 use crate::operation::{PinnedRelation, PinnedRelationList, Relation, RelationList};
-use crate::Validate;
 
 /// Enum of possible data types which can be added to the operations fields as values.
-#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
-#[serde(tag = "type", content = "value")]
+#[derive(Clone, Debug, PartialEq)]
 pub enum OperationValue {
     /// Boolean value.
-    #[serde(rename = "bool")]
     Boolean(bool),
 
     /// Signed integer value.
-    #[serde(rename = "int")]
     Integer(i64),
 
     /// Floating point value.
-    #[serde(rename = "float")]
     Float(f64),
 
     /// String value.
-    #[serde(rename = "str")]
-    Text(String),
+    String(String),
```

Comment:
Hm, I always thought it was so friendly that we call this "text" and not "string" and was hoping we could use "text" everywhere, also in schema field definitions. Do you think this is better?

Reply:
Ah, I understood that we like `String` more, this is why I changed it, it also reflects the `str` type definition maybe :-D But feel free to bring it back to `Text`, I don't have a preference :+1:

Reply:
Yeah, maybe "string" is actually less ambiguous, every hacker will know what it means. 

For "string" I always think about a short, maybe one-word value, and also I always wonder "does it support unicode?". For "text" I thought maybe it's more obvious that there is no, i don't know, 255 character limit and you can't put äëüö in it. But maybe that's also just me :)